wordpress-mobile/WordPress-iOS
# ULS: Use unified navigation bar

Ref #https://github.com/wordpress-mobile/WordPressAuthenticator-iOS/issues/182
Auth PR: https://github.com/wordpress-mobile/WordPressAuthenticator-iOS/pull/311

To test:

- Enable the `unifiedSiteAddress` feature flag.
- Run the app. Log out if necessary.
- Select Log In > Site Address.
- Verify the nav bar looks like this: (details are noted on the Auth PR)

<kbd><img width="470" alt="new_nav" src="https://user-images.githubusercontent.com/1816888/85800303-f3b61900-b6fd-11ea-8087-482348d4908b.png">
</kbd>

- Go to any other flow.
- Verify the nav bar looks like the old style, i.e:

<kbd><img width="470" alt="old_nav" src="https://user-images.githubusercontent.com/1816888/85800344-03356200-b6fe-11ea-94b7-cded6a98e72e.png"></kbd>

PR submission checklist:
